interface/main/tabs/menu/menus/standard.json

```diff
@@ -724,7 +724,7 @@
     "menu_id": "admimg",
     "children": [
       {
-        "label": "Globals",
+        "label": "EMR Config",
```

Comment:
I'd not use EMR but maybe App Config
Any other votes? @stephenwaite @bradymiller

Reply:
In my opinion, this is an EMR, so calling it an app feels weird. If this was built in Angular, i'd say it's an app. But since this can be ran on any web server and doesn't need Docker to run, it's more like a platform than an app.

Reply:
agree with Stephen, just Config/Configuration makes best sense. I just don't like EMR